src/one_dragon/base/push/push_config.py

```
@@ -0,0 +1,132 @@
+
+    def generate_channel_fields(self, channel_config_schemas: dict[str, list[PushChannelConfigField]]) -> None:
+        """
+        动态生成各个推送渠道的配置字段
+
+        Args:
+            channel_config_schemas: 各个渠道所需的配置字段
+
+        """
+        # 遍历所有配置组
+        for channel_id, field_list in channel_config_schemas.items():
+            channel_id_lower = channel_id.lower()
```

Ruff (0.14.0)

Local variable `channel_id_lower` is assigned to but never used

Remove assignment to unused variable `channel_id_lower`

(F841)